src/components/sections/Hero/Hero.tsx

```
@@ -1,54 +1,73 @@
-          <div className={styles.trustedBy}>
-            <p className={styles.trustedByLabel}>TRUSTED BY TEAMS AT</p>
-            <ul className={styles.companies}>
-              <li className={styles.company}>Acme Corp</li>
-              <li className={styles.company}>Global</li>
-              <li className={styles.company}>Nebula</li>
-              <li className={styles.company}>Vertex</li>
-              <li className={styles.company}>Horizon</li>
-            </ul>
+        {/* Social Proof / Trust Badges */}
+        <AnimatedElement animation="fadeIn" delay={400}>
+          <div className={styles.trustBadges} role="status" aria-live="polite">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)